src/engraving/tests/rhythmicgrouping_tests.cpp

```diff
@@ -103,3 +103,41 @@ TEST_F(Engraving_RhythmicGroupingTests, groupShortenNotes)
 {
     group("groupShortenNotes.mscx", "groupShortenNotes-ref.mscx");  // test for regrouping rhythms when notes should be shortened
 }
+
+TEST_F(Engraving_RhythmicGroupingTests, groupMaintainSelection)
+{
+    MasterScore* score = ScoreRW::readScore(RHYTHMICGRP_DATA_DIR + String::fromUtf8("groupSubbeats.mscx"));
```

Comment:
Or even
```suggestion
    MasterScore* score = ScoreRW::readScore(RHYTHMICGRP_DATA_DIR + String::fromUtf8("groupSubbeats.mscx"));
    ASSERT_TRUE(score);
```
because without score it doesn't make sense to continue the test execution.

Reply:
Thank's for the feedback! I've updated the PR to fix this.